ethcore/light/src/client/header_chain.rs

```diff
@@ -202,14 +202,21 @@ pub enum HardcodedSync {
 	Deny,
 }
 
+#[derive(MallocSizeOf)]
 /// Header chain. See module docs for more details.
 pub struct HeaderChain {
+	#[ignore_malloc_size_of = "perf"]
```

Comment:
If one doesn't know what the value assigned to `ignore_malloc_size_of` does (I didn't) then `"perf"` isn't a great help. Maybe `"ignored for perf reasons"` (or add a comment before the struct to explain: "skipping expensive allocation info here")